GPU/Vulkan/VulkanUtil.cpp

```diff
@@ -208,7 +210,7 @@ VkPipeline Vulkan2D::GetPipeline(VkRenderPass rp, VkShaderModule vs, VkShaderMod
 
 	VkPipelineColorBlendAttachmentState blend0 = {};
 	blend0.blendEnable = false;
-	blend0.colorWriteMask = 0xF;
+	blend0.colorWriteMask = depthStencilMode == VK2DDepthStencilMode::STENCIL_REPLACE_ALWAYS ? 0 : 0xF;
```

Comment:
For stencil upload, we want `VK_COLOR_COMPONENT_A_BIT`, no?  That's what GLES does.

-[Unknown]

Reply:
Oh, oops. Yes indeed.